Packages/com.unity.inputsystem/InputSystem/Devices/InputDevice.cs

```diff
@@ -519,7 +519,7 @@ protected virtual void OnRemoved()
         /// target="_blank">DeviceIoControl</a> on Windows and <a href="https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man2/ioctl.2.html"
         /// target="_blank">ioctl</a> on UNIX-like systems.
         /// </remarks>
-        public unsafe long ExecuteCommand<TCommand>(ref TCommand command)
+        public virtual unsafe long ExecuteCommand<TCommand>(ref TCommand command)
```

Comment:
Hmm, I remember this one led to compilation errors on PS4 back then. Was an il2cpp issue, though, so may have been fixed. Just remember that the particular combination of virtual + parameterized + ref parameter blew up il2cpp. An easy workaround, if we still need it, is to just have a separate, non-parameterized version that is virtual and just takes a direct pointer instead.

Reply:
Adjusted the code as per your suggestion